cgr/cgr.py

```diff
@@ -50,6 +50,19 @@ async def update_ratings_chameleon(self, ctx, players, winner):
         for player in players:
             await self.update_ratings_helper(ctx, [player], r2, 0.4, "chameleon")
 
+    async def update_ratings_blackjack(self, ctx, players, winners):
+        losers = set(players) - set(winners)
+        if (len(losers) > 0):
+            loser_rating = await self.get_average_rating(losers, "blackjack")
```

Comment:
`r2` should be a function of the _other team's rating_. Here you're just making it as if you're losing to your own team
`s` is a function of winning or losing, `1` for a win, `0`, for a loss, or something else for in between. Making it `0.4` every time means every game is treated like a tie